File: Programming Basics with Python/02 Conditional statements/More exercises/07_Flower_Shop.py
import math

flower_m = int(input())
flower_z = int(input())
flower_r = int(input())
cactus = int(input())
present = float(input())
profit = flower_m * 3.25 + flower_z * 4 + flower_r * 3.5 + cactus * 8
profit_with_taxes = profit * 0.95
result = profit_with_taxes - present
if profit_with_taxes >= present:
    print(f'She is left with {math.floor(result)} leva.')
else:
    print(f'She will have to borrow {math.ceil(abs(result))} leva.')

File: JS Basics/02 Conditional Statements - More Exercises/07. Flower Shop.js
function shop(magnolias, hyacinths, roses, cacti, gift) {
    let sold = Number(magnolias) * 3.25 + Number(hyacinths) * 4 + Number(roses) * 3.5 + Number(cacti) * 8;
    let profit = sold * 0.95;
    let result = Math.abs(gift - profit);
    if (Number(gift) > profit) {
        console.log(`She will have to borrow ${Math.ceil(result)} leva.`);
    }
    else {
        console.log(`She is left with ${Math.floor(result)} leva.`);
    }
}
shop(15, 7, 5, 10, 100)